content/post/debrick-core9g25-intro.md

```diff
@@ -0,0 +1,131 @@
++++
+title= "Debricking CORE 9G25: Understanding the boot process"
+date= 2018-05-20T11:02:29+02:00
+series = ["Unbrick CORE 9G25"]
+author = "Auke Willem Oosterhoff"
+description = "First post about fixing a bricked CORE 9G25 using SAM-BA."
+slug = "debricking-core-9g25-understanding-the-boot-process"
+draft= true
++++
+
+At [ACS Buildings][acs] one of our products is build around [CORE
+9G25][core9g25]. This System on a Module (SoM) has a the 400 MHz ARM CPU
+[AT91SAM9G25][at91sam9g25] from Atmel (which has been bought by Microchip). It
+also contains 256 MB of RAM and a NAND flash of the same size. The following
+image shows the SoM, it's taken from the website of [CoreWind][corewind].
+
+{{<figure src="/img/core9g25.jpg">}}
+
+The vendor doesn't ship the SoM with an empty NAND. It already contains 2
+bootloaders, a Linux kernel and a root filesystem. Because these programs are
+quite old we reflash the NAND and replace the stack with more recent versions
+of the programs. Reflashing the NAND is dangerous and can lead to bricked and
+unusable devices. In a series of 3 posts I'll demonstrate how to fix a bricked
+CORE 9G25. In this first post I'll explain how normal reflash procudere and
+boot procedure of the CORE 9G25 work. The second post will be about reflashing
+the NAND using the tool SAM-BA on Linux. The third post will be about the same
+thing, but then Windows is used to fix the NAND.
```

Comment:
` but then Windows is used to fix the NAND` to `but using Windows instead of Linux`